src/main/java/org/jenkinsci/plugins/liquibase/evaluator/ChangeSetDetail.java

```diff
@@ -23,7 +24,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(ChangeSetDetail.class);
 
     public static final int MAX_LINES = 15;
-    private List<Sql> sqls;
+    private List<String> sqls;
```

Comment:
be careful with such changes, because the object might be stored on the disk (JEP-200 is a risk also!). In such case it is easier to introduce a new field and add some migration logic.


```
private transient List<Sql> sqls;
private List<String> sqlIDs;

private Object readResolve() {
}
```

or so...